config/environments/production.rb

```diff
@@ -12,6 +12,8 @@
   config.active_record.dump_schema_after_migration = false
   config.action_mailer.smtp_settings = JSON.parse(Figaro.env.smtp_settings).symbolize_keys
 
+  routes.default_url_options[:protocol] = 'https'
```

Comment:
Unsure, this is for the router, and that's for the mailer...maybe the router encompasses the mailer?